supportedBackends/google/pipelines/common/src/test/scala/cromwell/backend/google/pipelines/common/PipelinesApiAttributesSpec.scala

```diff
@@ -55,6 +55,14 @@ class PipelinesApiAttributesSpec extends FlatSpec with Matchers {
 
   }
 
+  it should "parse localization-attempts" taggedAs IntegrationTest in {
+    val backendConfig = ConfigFactory.parseString(configString(genomics = "localization-attempts = 31380"))
+
+    val jesAttributes = PipelinesApiAttributes(googleConfig, backendConfig)
+    jesAttributes.localizationConfiguration.localizationAttempts.value should be(31380)
```

Comment:
ToL: This is copy/paste, but we should stop using "JES". Same goes for other instances in this PR. I'd be ok w/ separate ticket/PR to clean up the remaining references.